components/tabs/tabs.js

```diff
@@ -780,6 +777,19 @@ class Tabs extends LocalizeCoreElement(ArrowKeysMixin(RtlMixin(FocusVisiblePolyf
 		return (position > measures.tabsContainerRect.width - scrollButtonWidth) && (position < measures.tabsContainerRect.width);
 	}
 
+	_resetFocusables() {
+		const selectedTab = this._tabInfos.find(ti => ti.selected);
+		if (selectedTab) this._setFocusable(selectedTab);
+		this.requestUpdate();
+	}
+
+	_setFocusable(tabInfo) {
+		const currentFocusable = this._tabInfos.find(ti => ti.activeFocusable);
+		if (currentFocusable) currentFocusable.activeFocusable = false;
+
+		tabInfo.activeFocusable = true;
+	}
```

Comment:
I broke this out into a helper function because it was being used all over the place.